src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String name) {
+        super(name);
+        radius = new RandomNumberGenerator().getRandomProperties();
```

Comment:
```suggestion
    public Circle(String name, String color, double radius) {
        super(name, color);
        this.radius = radius;
```

you should generate random values in FigureGeneratorSupplier and just pass them to the constructor
same for other figures